interface/src/CrashHandler_Crashpad.cpp

```diff
@@ -70,8 +76,99 @@ LONG WINAPI vectoredExceptionHandler(PEXCEPTION_POINTERS pExceptionInfo) {
         client->DumpAndCrash(pExceptionInfo);
     }
 
+    if (pExceptionInfo->ExceptionRecord->ExceptionCode == 0xE06D7363) {
```

Comment:
I can likely make it a constant somewhere, was kinda split on that as this would be the only place it would be used.

Reply:
Right, what does `0xE06D7363` stand for? Surely this has some name to it.

Also it's in the coding standard.

Reply:
as far as I can tell it means "Microsoft C++ Exception".

I tried to take an ascii chart to it in case it came up with a word but I didn't find anything.